fair_mango/metrics/metrics.py

```diff
@@ -34,11 +35,13 @@ def __init__(
         real_target: Sequence[str] | None = None,
         predicted_target: Sequence[str] | None = None,
         positive_target: Sequence[int | float | str | bool] | None = None,
+        label: str = "result",
     ):
```

Comment:
I see no documentation. Documentation should be append along with the code, not after

Reply:
The documentation is on another branch that is WIP. 
The new functions I am implementing are documented directly. Like the `super_set_fairness_metrics` and `super_set_performance_metrics` functions added in this PR.